code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -37,12 +37,12 @@
 /mob/living/simple_animal/hostile/syndicate/Die()
 	..()
 	if(corpse)
-		new corpse (src.loc)
+		corpse.createCorpse()

```

Comment:
The issue I have with it is that createCorpse references src, which is an issue if you don't actually new the landmark at any point.